grafana-dash.json

```diff
@@ -224,6 +224,50 @@
         "tagsQuery": "",
         "type": "query",
         "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "${DS_PROMETHEUS}",
+        "definition": "label_values(fail2ban_banned_ip, instance)",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Host",
+        "multi": false,
```

Comment:
If you can change `includeAll` and `multi` to `true`, I'll gladly pull this. I prefer to see a total of all my instances and this way we can add a feature without removing another.